src/main/java/com/wiley/autotest/selenium/elements/upgrade/v3/NullShouldImmediately.java

```diff
@@ -0,0 +1,60 @@
+package com.wiley.autotest.selenium.elements.upgrade.v3;
+
+import com.wiley.autotest.selenium.SeleniumHolder;
+import com.wiley.autotest.selenium.context.SearchStrategy;
+import com.wiley.autotest.selenium.elements.upgrade.TeasyElement;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+
+import java.util.function.Function;
+
+/**
+ * Created by vefimov on 26/04/2017.
+ */
+public class NullShouldImmediately implements Should {
+
+    private TeasyElement element;
+
+    public NullShouldImmediately(TeasyElement element) {
+        this.element = element;
+    }
+
+    //null element can't be displayed
+    public void beDisplayed() {
+        throwException();
+    }
+
+    //null element is absent by default
+    public void beAbsent() {
+        doNothing();
+    }
+
+    public void haveText(String text) {
+        throwException();
+    }
+
+    public void haveAttribute(String attributeName, String value) {
+        throwException();
+    }
+
+    public void haveAttribute(String attributeName) {
+        throwException();
+    }
+
+    public void notHaveAttribute(String attributeName) {
+        throwException();
+    }
+
+    public void customCondition(Function<WebDriver, Boolean> condition) {
+        new TeasyFluentWait<>(SeleniumHolder.getWebDriver(), new SearchStrategy(0)).waitFor(condition);
```

Comment:
Could you change this magic number?

Reply:
I have mixed thoughts about it. From one point of view I can make it a constant or var, but at the same time, there will appear an extra line which will not make it much more clear? Isn't it clear enough that we create a strategy with 0 timeout here? (i mean it's quite deep inner logic)